src/main/java/org/apache/mesos/specification/TaskSpecification.java

```diff
@@ -38,8 +38,17 @@
     Collection<VolumeSpecification> getVolumes();
 
     /**
-     * Returns the placement constraint for this task type. This can be anything from "colocate with
-     * another type", "ensure instances are only launched on nodes with attribute X", etc. This may
+     * Returns the Configuration Files to be written for this task, or an empty list if no custom
+     * configurations should be written.
+     *
+     * Note: There is a limit of 100KB (102,400B) across all template data for a given task. If you
```

Comment:
I don't see anywhere that we're actually enforcing this. Is this threshold set by `TaskInfo` by default?